vars/buildPlugin.groovy

```diff
@@ -279,10 +279,14 @@ def call(Map params = [:]) {
                    * plugins' PR builds could not be consumed by anything else anyway, and all
                    * plugins currently in the BOM are incrementalified.
                    */
-                  if (incrementals && currentBuild.currentResult == 'SUCCESS') {
+                  if (incrementals && currentBuild.currentResult == 'SUCCESS' && useContainerAgent) {
                     withCredentials([string(credentialsId: 'launchable-jenkins-bom', variable: 'LAUNCHABLE_TOKEN')]) {
-                      launchable('verify')
-                      launchable('record commit')
+                      if (isUnix()) {
+                        sh 'launchable verify && launchable record commit'
+                      } else {
+                        // TODO launchable.exe still not working for some reason
```

Comment:
https://github.com/jenkins-infra/helpdesk/issues/3484#issuecomment-1552280288

Reply:
Should be addressed by https://github.com/jenkins-infra/docker-inbound-agents/pull/83

Reply:
https://ci.jenkins.io/job/Plugins/job/text-finder-plugin/job/PR-172/24/console is still failing, but I am not sure whether that is because https://github.com/jenkins-infra/docker-inbound-agents/pull/83 failed to fix the problem or simply has not been deployed to production yet.

Reply:
It wasn't deployed yet, done in https://github.com/jenkins-infra/jenkins-infra/pull/2845

Confirmed working in https://ci.jenkins.io/job/Plugins/job/text-finder-plugin/job/PR-172/25/console